src/Screens/landing/style.css

```diff
@@ -1,52 +1,60 @@
+.top-container {
+    background-color: var(--themeYellow);
+}
+
+.header-nav {
+    display: flex;
+    justify-content: flex-end;
+}
+
 .top-section, .bottom-section {
-    height: 50%;
-    /* risky */
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    align-items: center;
 }
 
 .top-section {
-    background-color: var(--yellow);
+    /* background-color: var(--themeYellow); */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 22em;
+}
+
+.top-subsection {
+    /* margin-top: 30px; */
+    width: 70%;
 }
 
 .name-container {
-    border: 12px solid var(--blue);
+    border: 12px solid var(--themeBlue);
     border-radius: 12px;
-    width: 70%;
-    position: absolute;
     top: 25px;
     text-align: center;
 }
 
 .name {
     /* color: #3C465F; */
-    color: var(--blue);
+    color: var(--themeBlue);
     font-size: 3em;
+    font-family: 'Maven-pro-bold';
     margin: 0.4em 0;
 }
 
 .headline {
-    position: absolute;
-    top: 19%;
-    color: var(--blue);
-}
-
-.code-symbol {
-    position: absolute;
-    top: 23.5%;
-    color: var(--blue);
+    margin-top: 1.5em;
+    color: var(--themeBlue);
+    text-align: center;
 }
-
 .image-container {
     background-color: white;
     height: 240px;
```

Comment:
have you tested this on responsive ?

I think it is better if we can use `em` or `rem` for this.

Or decrease height and width in media screens

Reply:
yes, it will be decreased in media queries.